app/components/ui/atoms/PersonInfo.md

```diff
@@ -0,0 +1,15 @@
+```js
+<PersonInfo
+  person={{
+    id: '1234',
+    name: 'Michael J Frank',
+    aff: 'Max Planck Institute, Germany',
+    focuses: [
+      'genetics of global gene expression',
+      'mouse models of human disease',
+      'haematology',
+    ],
+    expertises: ['Neuroscience', 'Cell Biology'],
+  }}
+/>
+```
```

Comment:
This seems to be failing on the styleguide

Reply:
<img width="379" alt="screen shot 2018-11-27 at 12 14 56" src="https://user-images.githubusercontent.com/2573277/49081245-1ccea900-f23e-11e8-90f0-6ac0b4bec8fd.png">